src/tulip/infos.ts

```diff
@@ -258,6 +347,139 @@ infos = class InstanceTulip {
       serumMarket,
     };
   }
+
+  private static _parseOrcaVault(data: any, vaultId: PublicKey): types.OrcaVault {
+    const decodeData = ORCA_VAULT_LAYOUT.decode(data);
+    const { base, farmData } = decodeData;
+
+    const vaultAccount = vaultV2Config.vaults.accounts.find((account) => account.orca?.account == vaultId.toString());
```

Comment:
What is `vaultV2Config` used for here? I see its imported from `ids` but it seems to keep `vaults.accounts`

Reply:
Add `feeCollector` from `vaultV2Config` this is an account need to use in ix, but didn't write in on-chain account.

Reply:
Oh yes I recalled. I think the word `vault` is kinda abused here. How about renaming:

- File name: ~~`ids/vault.ts`~~ => `ids/config.ts`
- Object name: ~~`vaultV2Config`~~ => `configV2`  ([ref](https://github.com/DappioWonderland/navigator/blob/tulip-vault/src/tulip/ids/vault.ts#L3))

In this case, the chain of invoking becomes: ~~`vaultV2Config.vaults.accounts`~~ => `configV2.vaults.accounts` which is simpler